### DISCUSSION TITLE
Keyevents are not identified in search mode

### DISCUSSION THREAD

**Discussion by elferherrera** [helix-editor/helix#10415] [General]
### Summary

After selecting text and trying to perform a search, there are keyevents that are not captured by `Helix`. I have identified that these keyevents are those produced by the combination `alt gr + <key>`.

This is an issue that we used to have in Reedline. To fix it we had to add [these options](https://github.com/nushell/reedline/blob/main/src/edit_mode/emacs.rs#L110) to the event processing stage

### Reproduction Steps

Using a non US keyboard try to type [, ], \\, in search mode

### Platform

Windows

### Terminal Emulator

windows terminal

### Helix Version

latest main

**Comment by archseer** [comment]
I recently added this workaround for windows https://github.com/helix-editor/helix/commit/764adbdcf6f0a5615d1ddd3abb7ed7b9cf0f2d92

Could be that it's not working in pickers?
